File: 16_confirm_ending.py
# confirm the ending of a string

def confirmEnding(str, target):
#    if(str.endswith(target)):
#        return True
#    return False
# using sub string
    if(str[-len(target):] == target):
        return True
    return False



print(confirmEnding('Bastian', 'n'))
print(confirmEnding('Bastian', 'on'))

File: 16_confirm_ending.js
// confirm the ending of a string

function confirmEnding(str, target){
//   if(str.endsWith(target)){
//        return true;
//    }
//    return false;

// using substr() function
    if(str.substr(-target.length) === target){
        return true;
    }
    return false;
}



console.log(confirmEnding("Bastian", "n"));
console.log(confirmEnding("Bastian", "no"));
